pkg/networking/backend_sg_provider.go

```diff
@@ -267,3 +332,7 @@ func isEC2SecurityGroupNotFoundError(err error) bool {
 	}
 	return false
 }
+
+func getObjectKey(obj metav1.Object) string {
+	return reflect.TypeOf(obj).String() + "/" + k8s.NamespacedName(obj).String()
```

Comment:
maybe pass the gvk of object in `get` and `release` explicitly?

Reply:
refactored to pass the type string